<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dmfUploadDialog</class>
 <widget class="QDialog" name="dmfUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMF Simulation Upload</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add/Update DMF</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Simulation Name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Application:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="sinterConfigGUIButton">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Edit or create a new SimSinter configuration file with the Sinter Config GUI.  This is a windows only option.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Create/Edit</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/icons/edit.svg</normaloff>:/icons/icons/edit.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="configFileButton">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>106</red>
               <green>104</green>
               <blue>100</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <underline>false</underline>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Search for a sinter configuration file.</string>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/icons/search.svg</normalon>
          </iconset>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="appEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Application used to run the simulation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sinter Configuration File:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="simNameEdit">
       <property name="toolTip">
        <string>Name of simulation on Turbine.  The drop down will show and allow the user to select existing simulations.  A new name can also be entered to create a new simulation.</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="clearTableButton">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Additional Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="addFileButton">
          <property name="text">
           <string>Add Files...</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/icons/add.svg</normaloff>:/icons/icons/add.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeFileButton">
          <property name="text">
           <string>Remove Files</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/icons/remove.svg</normaloff>:/icons/icons/remove.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="relpathButton">
          <property name="text">
           <string>Resource Relative Path</string>
          </property>
          <property name="icon">
           <iconset resource="../icons.qrc">
            <normaloff>:/icons/icons/subArrow.svg</normaloff>:/icons/icons/subArrow.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="fileTable">
        <column>
         <property name="text">
          <string>Resource</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
     <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
	 <widget class="QLabel" name="label">
	   <property name="text">
             <string>DMF Repo:</string>
	   </property>
	 </widget>
       </item>
       <item row="0" column="1">
	 <widget class="QLineEdit" name="dmfRepo">
	   <property name="enabled">
             <bool>false</bool>
	   </property>
	   <property name="toolTip">
             <string>DMF Repo</string>
	   </property>
	 </widget>
       </item>
       <item row="0" column="2">
	 <widget class="QPushButton" name="selectDMFRepoButton">
	   <property name="toolTip">
             <string>Select a DMF repo</string>
	   </property>
	   <property name="text">
             <string>SelectDMF</string>
	   </property>
	 </widget>
       </item>
     </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Okay</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/check.svg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/exit.svg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="foqus_lib.gui.icons.qrc"/>
 </resources>
 <connections/>
</ui>
